### PR TITLE
#15 add drupal-check

### PR DESCRIPTION
**Pull request by wallon-ines**
Add Drupal check deprecation for phpstan

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,8 +17,9 @@
     "drupal/coder": "^8.2.12",
     "jakub-onderka/php-parallel-lint": "^1.0",
     "localheinz/composer-normalize": "^1.2",
+    "sebastian/phpcpd": "^3.0",
+    "mglaman/drupal-check": "^1.0"
```
Comment (Eli-TW) [inline composer.json:21]: Isn't this now invalid json?

**Comment by vijaycs85** [comment]
This project aim to provide coverage for individual contrib modules as well as a site. but with module case, I faced issues around the drupal bootstrap. so I am skipping this for now.

**Comment by MPParsley** [comment]
drupal-check requires drupal to be installed.

**Comment by vijaycs85** [comment]
I am not sure if we are going to manage it here. Feel free to update your config once installed.

**Pull request closed without merging**